pubg_python/mixins.py

```diff
@@ -37,7 +38,7 @@ def offset(self, value):
         self.endpoint.args['page[offset]'] = value
         return self
 
-    def next(self):
+    def __next__(self):
```

Comment:
I don't think `next` should be a magic method since the iterator returned by a QuerySet iterates the records fetched from the API and not the pages, what do you think?